### PR TITLE
Fully describe call to non-configured method on strict Fake

### LINKED ISSUE #492: Showing method call arguments in ExpectationException message
Hi, FakeItEasy owner. Thanks for making such a great Framework.

I have a feature request for strict mode. When it fails on ExpectationException, besides showing the calling method name in the exception message, could you please showing the calling arguments?

That information is so important and much useful for debugging.

### PR DESCRIPTION
**Pull request by blairconrad**
Fixes #492.

### CODE REVIEW

File: tests/FakeItEasy.Tests/FakeOptionsExtensionsTests.cs [hunk 1]
```diff
@@ -3,28 +3,12 @@ namespace FakeItEasy.Tests
...
-            // Act
-            var exception = Record.Exception(() => foo.Bar());
-
-            // Assert
-            exception.Should()
-                .BeAnExceptionOfType<ExpectationException>()
-                .WithMessage("Call to non configured method \"Bar\" of strict fake.");
-        }
-
-        [Fact]
```
Comment (thomaslevesque) [inline tests/FakeItEasy.Tests/FakeOptionsExtensionsTests.cs]: Was this test removed because it's redundant with the new specs?

Comment (blairconrad) [inline tests/FakeItEasy.Tests/FakeOptionsExtensionsTests.cs]: Sorry, I should've said. The new specs cover it completely. And it's easier to remove than update.

**Comment by blairconrad** [comment]
> Thanks @blairconrad, this looks very good!

Thanks. I enjoyed using the new `ExceptionMessages`.

File: tests/FakeItEasy.Tests/FakeOptionsExtensionsTests.cs (hunk 1 above)
Comment (thomaslevesque) [inline tests/FakeItEasy.Tests/FakeOptionsExtensionsTests.cs]: > And it's easier to remove than update.

👍

**Comment by blairconrad** [comment]
Thanks, @thomaslevesque!

**Pull request merged**